app/presenters/result_set_presenter.rb

```diff
@@ -77,6 +69,11 @@ def highlight_top_result?
       best_bet?
   end
 
+  def highlight(index)
+    index === 0 && highlight_top_result?
+  end
```

Comment:
If this method is needed only by `SearchResultPresenter`, could we move it in there?

Reply:
It's called on line 39 of the same file and depends on another method right next to it, which itself depends on a number of other variables. It seemed tidier to leave it here than trying to pass everything through to `SearchResultPresenter`, but if there's a way of doing that nicely, I'm up for changing it, sure.